src/reprinter/index.ts

```diff
@@ -14,6 +16,19 @@ export class Reprinter {
     new JsonReprinter()
   ];
   public static rewriteFile(filename: string, options: ReprinterOptions) {
+    const ignoreFilePath = path.resolve(".sortierignore");
+    try {
+      let ignoreText = FileUtils.readFileContents(ignoreFilePath).trim();
+      const relativeFilePath = path.relative(path.resolve("."), filename);
+      if (0 < ignoreText.length) {
+        let ig = ignore();
+        ig.add(ignoreText.split(/\r?\n/));
```

Comment:
Could also filter out comments here:

```suggestion
        ig.add(ignoreText.split(/\r?\n/).filter((word) => !word.match(/^\s*#/));
```

Reply:
Comments are already handled by the ignore package
https://www.npmjs.com/package/ignore#addpatterns-arraystring--ignore-this

Reply:
Ah, nice